### ISSUE TITLE
visualization of sensors in VPL

### ISSUE THREAD

**Issue by riedo** [aseba-community/aseba#397]
As discussed with Stéphane and Didier Roy: it would be nice, in VPL, to visualize the current value of a sensor (like the memory window in studio)
It could help to choose where to place the sliders for IR sensors for example.

**Comment by stephanemagnenat** [comment]
Indeed, and the interaction design for this is an interesting question. I think that this shall be developed in the context of Thymio AR on tablet.
